Makefile

```diff
@@ -47,6 +48,7 @@ BUILD_IMAGE 	?= us.gcr.io/k8s-artifacts-prod/build-image/kube-cross:v$(GO_VERSIO
 ISO_BUILD_IMAGE ?= $(REGISTRY)/buildroot-image
 KVM_BUILD_IMAGE ?= $(REGISTRY)/kvm-build-image:$(GO_VERSION)
 
+KIND_BASE_IMAGE_GCR ?= $(REGISTRY)/kindbase:$(KIND_VERSION)
```

Comment:
this probably makes it more confusing

Reply:
As I mentioned above, it was a refactoring base

Reply:
Trying do all changes at once (like in the previous PR), makes it much harder to test - or see what happened.

Hard enough to see what changed between the KIND entrypoint and the new KIC entrypoint, if you ask me ?